pkg/kubernetes/selinux.go

```diff
@@ -10,10 +10,10 @@ import (
 	"github.com/suse-edge/edge-image-builder/pkg/image"
 )
 
-func SELinuxPackage(version string) (string, error) {
-	const (
-		k3sPackage  = "k3s-selinux"
-		rke2Package = "rke2-selinux"
+func SELinuxPackage(version string, sources *image.ArtifactSources) (string, error) {
+	var (
+		k3sPackage  = sources.Kubernetes.K3s.SelinuxPackage
```

Comment:
Just use the value directly, there's no point in setting these variables anymore.